jsx/Breadcrumbs.js

```diff
@@ -14,25 +16,104 @@
  */
 class Breadcrumbs extends React.Component {
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      displayCount: 0
+    };
+
+    this.checkScreenSize = this.checkScreenSize.bind(this);
+  }
+
+  componentWillMount() {
+    this.checkScreenSize();
+    if (typeof window !== 'undefined')
+      window.addEventListener('resize', this.checkScreenSize);
+  }
+
+  componentWillUnmount() {
+    if (typeof window !== 'undefined')
+      window.removeEventListener('resize', this.checkScreenSize);
+  }
+
+  checkScreenSize() {
+    // Used to check to current window size and
+    // sets the number of breadcrumbs to show
+    var windowWidth = window.innerWidth;
+    var displayCount = 4;
+
+    if (windowWidth <= 500) {
```

Comment:
I don't think we should rely on window width to determine number of breadcrumbs to show, since the breadcrumb width will always differ depending on the text length.

If there's absolutely no other way, I would at least use sizes used in bootstrap breakpoints http://getbootstrap.com/css/#grid-options